internal/resource/nvml-device.go

```diff
@@ -86,3 +90,37 @@ func (d nvmlDevice) GetTotalMemoryMB() (uint64, error) {
 	}
 	return info.Total / (1024 * 1024), nil
 }
+
+func (d nvmlDevice) GetDisplayMode() (string, error) {
+	info, retVal := d.Device.GetPciInfo()
```

Comment:
What is the difference between this logic and the value returned by `nvmlDeviceGetDisplayMode` for the device?

Reply:
`nvmlDeviceGetDisplayMode` method indicates whether a physical display (like monitor) is currently connected to any of the device's connectors